pandas/core/arrays/sparse.py

```diff
@@ -2205,10 +2206,8 @@ def to_coo(self):
         float32. By numpy.find_common_type convention, mixing int64 and
         and uint64 will result in a float64 dtype.
         """
-        try:
-            from scipy.sparse import coo_matrix
-        except ImportError:
-            raise ImportError('Scipy is not installed')
+        import_optional_dependency("scipy")
```

Comment:
wouldn't you write this as

```
scipy = import_optional_dependency('scipy')
from ....
```

to make this explicit?

Reply:
If you're using something from the top-level of the package, then yes. But we want at `scipy.sparse` which isn't imported by default. And we don't using anything from `scipy`'s top-level, so we would need

```
scipy = import_optional_dependency('scipy')  # noqa
```